rubin_scheduler/scheduler/example/example_scheduler.py

```diff
@@ -517,25 +523,26 @@ def gen_long_gaps_survey(
             u_template_weight=u_template_weight,
             g_template_weight=g_template_weight,
             blob_names=blob_names,
+            u_exptime=u_exptime,
+            nexp=nexp,
         )
         scripted = ScriptedSurvey(
-            [bf.AvoidDirectWind(nside=nside), bf.AltAzShadowMaskBasisFunction(nside=nside, shadow_minutes=0)],
+            [bf.AvoidDirectWind(nside=nside)],
```

Comment:
How does the ScriptedSurvey for the followup triplets know what parts of the sky are unavailable for pointing due to the telescope?

Reply:
There's a `_check_alts_ha` method on `ScriptedSurvey`. It doesn't make as much sense to use HEALpix maps when there's already a list of RA,dec positions.